Index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="./CSS/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <div class="header-text-outer">
+          <div class="header-text-inner">
+              <span>YOUR NAME</span>

```

Comment:
Whenever you want to display something in uppercase then make it that string upper case using text-transform css property.

Reply:
done with this